website/content/terminal/guides/intros/portfolio/po.md

```diff
@@ -377,7 +377,7 @@ Further analysis can be done by looking at the portfolio's returns <a href="http
 
 ````
 2022 May 31, 03:39 (🦋) /portfolio/po/ $ plot HRP_0 -ct SECTOR -hi -dd
-```
+````
 
```

Comment:
Shouldn't they all be 3 ticks?

Reply:
Whether it's 3 or 4 doesn't really matter, just needs to be consistent which it wasn't.